components/JoinUs.js

```diff
@@ -0,0 +1,79 @@
+import styled from 'styled-components'
+
+const JoinUsContainer = styled.div`
+  display: flex;
+  flex-direction: column;  
+  width: 750px;  
+  padding: 160px 0 158px 81px;
+`;
+
+const Title = styled.span`
+  background: ${p => p.theme.colors.primaryGradient};
+  -webkit-background-clip: text;
+  -moz-background-clip: text;
+  -webkit-text-fill-color: transparent; 
+  -moz-text-fill-color: transparent;
+
+  font-size: 43px;
+  font-weight: bold;
+  margin-bottom: 42px;
+`;
+
+const ButtonContainer = styled.div`
+  display: flex;
+  gap: 42px;
+  align-items: center;  
+`;
+
+const RegularText = styled.span`
+  line-height: 22px;
+  font-size: 18px;
+  color: ${p => p.theme.colors.bodyText};
+`;
+
+const AccentedText = styled.span`
+  line-height: 22px;
+  font-size: 18px;
+  background: ${p => p.theme.colors.primaryGradient};
+  -webkit-background-clip: text;
+  -moz-background-clip: text;
+  -webkit-text-fill-color: transparent; 
+  -moz-text-fill-color: transparent;
+`;
```

Comment:
I recently added a few typography components in Typography.js - can you see if any of them suits your needs? If more typography components need to be added, I'd suggest adding them in Typography.js. 
I'd suggest looking at the design specs for the exact type (i.e. the body paragraph below should use `Body` in Typography.js)
<img width="909" alt="Screen Shot 2021-08-17 at 10 11 14 AM" src="https://user-images.githubusercontent.com/38872354/129770663-02227fb7-95ba-4ee8-82b1-3ceb2f4d0b21.png">
For the 'Join Us' heading, the text isn't named, but I would take a look at the design specs for each Typography element and see which element it is. If it's missing in the current Typography.js, you can just add it as a new component.
<img width="933" alt="Screen Shot 2021-08-17 at 10 11 37 AM" src="https://user-images.githubusercontent.com/38872354/129770716-3fa4a312-324a-4c48-a2cc-232cd74eea3a.png">
<img width="718" alt="Screen Shot 2021-08-17 at 10 11 52 AM" src="https://user-images.githubusercontent.com/38872354/129770754-f1b6f3c7-c0da-41cd-88ab-95eea4f6a0ed.png">

I've also opened a new issue for typography #48 - would be great if you can check off the missing element implemented once it's added!

Reply:
Thank you for making me aware of the new typography components! Will replace them with what you have got there